### PR TITLE
document populationPedCreating

### PR DESCRIPTION
**Pull request by PichotM**

### CODE REVIEW

File: content/scripting-reference/_index.md [hunk 1]
```diff
@@ -8,6 +8,6 @@ weight: 500
 - [Client functions](/scripting-reference/client-functions)
 - [Server functions](/scripting-reference/server-functions)
 <!-- TODO: - [Useful functions](/scripting-reference/useful-functions)-->
-- [Client events](/scripting-reference/client-events)
-- [Server events](/scripting-reference/server-events)
+- [Client events](/scripting-reference/events/client-events)
```
Comment (blattersturm) [inline content/scripting-reference/_index.md]: Shouldn't this be a separate PR, perhaps?

File: content/scripting-reference/events/list/populationPedCreating.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+---
+title: populationPedCreating
+weight: 551
+---
+
+This event is fired after population creation and allows manipulating population from script.
```
Comment (blattersturm) [inline content/scripting-reference/events/list/populationPedCreating.md]: Before, not after. Rather phrase it as "fired before creating a population ped".

File: content/scripting-reference/events/list/populationPedCreating.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+---
+title: populationPedCreating
+weight: 551
+---
+
+This event is fired after population creation and allows manipulating population from script. 
+You can use setPosition and setModel.
```
Comment (blattersturm) [inline content/scripting-reference/events/list/populationPedCreating.md]: Weird sentence, rather remove and define what `setters` contains after it.

File: content/scripting-reference/events/list/populationPedCreating.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+```
+int posX, int posY, int posZ, string model, object setters
+```
+
+- **posX**: X spawn position.
+- **posY**: Y spawn position.
+- **posZ**: Z spawn position.
+- **model**: Spawn model.
+- **setters**: The source object of the ped, that can be used with a limited set of functions only.
```
Comment (blattersturm) [inline content/scripting-reference/events/list/populationPedCreating.md]: There's no such thing as a `ped source object`, it's literally just an object to override what the ped spawns as. The fields in it should be documented here, as well.

File: content/scripting-reference/events/list/populationPedCreating.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+You can use setPosition and setModel.
+
+Parameters
+----------
+
+```
+int posX, int posY, int posZ, string model, object setters
+```
+
+- **posX**: X spawn position.
```
Comment (blattersturm) [inline content/scripting-reference/events/list/populationPedCreating.md]: "The X position the ped is going to spawn at", or so. Same for Y/Z.

File: content/scripting-reference/events/list/populationPedCreating.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+- **posY**: Y spawn position.
+- **posZ**: Z spawn position.
+- **model**: Spawn model.
```
Comment (blattersturm) [inline content/scripting-reference/events/list/populationPedCreating.md]: "The model the ped is going to spawn as"?

File: content/scripting-reference/events/list/populationPedCreating.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
+---
+title: populationPedCreating
+weight: 551
+---
+
+This event is fired after population creation and allows manipulating population from script. 
+You can use setPosition and setModel.
+
```
Comment (blattersturm) [inline content/scripting-reference/events/list/populationPedCreating.md]: No mention of this event being cancelable, since, you know, it is.

**Comment by PichotM** [comment]
*updated*
